pubsub/azure/servicebus/metadata.go

```diff
@@ -22,8 +22,8 @@ type metadata struct {
 	HandlerTimeoutInSec             int    `json:"handlerTimeoutInSec"`
 	LockRenewalInSec                int    `json:"lockRenewalInSec"`
 	MaxActiveMessages               int    `json:"maxActiveMessages"`
-	MaxReconnectionAttempts         int    `json:"maxReconnectionAttempts"`
-	ConnectionRecoveryInSec         int    `json:"connectionRecoveryInSec"`
+	MaxConnectionRecoveryInSec      int    `json:"maxConnectionRecoveryInSec"`
```

Comment:
This is a breaking change - see https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-servicebus/